src/containers/App/index.tsx

```diff
@@ -104,6 +104,10 @@ export const AppWrapper = () => {
                   path={updatePath('/ledgers')}
                   element={<Navigate to={updatePath('/')} replace />}
                 />
+                <Route
+                  path={updatePath('/index.html')}
+                  element={<Navigate to={updatePath('/')} replace />}
+                />
                 {/* End: Redirects */}
```

Comment:
You may also want to add redirect for `/index.htm` since some servers still use it.

Reply:
added